.eleventy.js

```diff
@@ -153,6 +153,24 @@ module.exports = function (config) {
     return memoize(collection.getAll());
   });
 
+  // Filters through all collection items and finds content that has
+  // CSS_ORIGIN set to 'next'. This allows shortcodes to determine if we
+  // are in a design system context or a legacy context
+  config.addCollection('designSystemGlobals', (collection) => {
+    global.__designSystemPaths = [
+      ...new Set(
+        collection
+          .getAll()
+          .filter(({data}) => data.CSS_ORIGIN === 'next')
+          .map(({filePathStem}) => filePathStem),
+      ),
+    ];
+
+    global.__designSystemComponents = require('./src/site/_data/design/components');
```

Comment:
Just make your code require this when it needs it. Unless I'm missing something and this is trying to work around 11ty's reload semantics. In which case this should be commented anyway 🤷

Reply:
This is going to go now. It was just a test-case that was janky anyway.